frontend/device/sdl/device.lua

```diff
@@ -120,6 +120,13 @@ local Desktop = Device:extend{
     hasExitOptions = notOSX,
 }
 
+local Flatpak = Device:extend{
+    model = "Flatpak",
+    isDesktop = yes,
+    canRestart = no,
```

Comment:
The flatpak version can't restart?

Reply:
That code, the handling of `$@/$ARGS` does not look right to me:
```bash
▸ set -- 1 2 3
▸ echo "${#@}"
3
▸ ARGS="$@"
▸ echo "$ARGS"
1 2 3
▸ printf '“%s”\n' "$ARGS"
“1 2 3”
▸ ARGS=""
▸ printf '“%s”\n' "$ARGS"
“”
```

Reply:
It works nonetheless :).
Feel free to suggest something better.

Reply:
wow, I see you're fast :)